lochness/rpms/__init__.py

```diff
@@ -227,13 +228,16 @@ def sync(Lochness, subject, dry=False):
         else:
             latest_pull_mtime = 0
 
+        if len(source_df) == 0:  # do not save if the dataframe is empty
+            continue
+
         # if last_modified date > latest_pull_mtime, pull the data
         source_df['LastModifiedDate'] = pd.to_datetime(
                 source_df['LastModifiedDate'])
         if not source_df['LastModifiedDate'].max() > \
                 pd.to_datetime(latest_pull_mtime):
             print('No new updates')
-            break
+            continue
```

Comment:
Can you post a snapshot of `subject_df_dict`? Otherwise, I cannot undestand the `break-->continue` migration.

Reply:
Can you post a snapshot of `subject_df_dict`? One that activates this if block maybe? Otherwise, I cannot understand the `break-->continue` migration.